src/common/com/intellij/plugins/haxe/ide/quickfix/CreateGetterSetterQuickfix.java

```diff
@@ -0,0 +1,90 @@
+package com.intellij.plugins.haxe.ide.quickfix;
+
+import com.intellij.codeInsight.intention.impl.BaseIntentionAction;
+import com.intellij.openapi.application.ApplicationManager;
+import com.intellij.openapi.command.WriteCommandAction;
+import com.intellij.openapi.editor.Editor;
+import com.intellij.openapi.project.Project;
+import com.intellij.plugins.haxe.ide.generation.GetterSetterMethodBuilder;
+import com.intellij.plugins.haxe.lang.psi.HaxeNamedComponent;
+import com.intellij.plugins.haxe.model.HaxeClassModel;
+import com.intellij.plugins.haxe.model.HaxeFieldModel;
+import com.intellij.plugins.haxe.util.HaxeElementGenerator;
+import com.intellij.psi.PsiElement;
+import com.intellij.psi.PsiFile;
+import com.intellij.psi.PsiParserFacade;
+import com.intellij.util.IncorrectOperationException;
+import org.jetbrains.annotations.Nls;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.List;
+
+/**
+ * @author Ilya Malanin
+ */
+
+public class CreateGetterSetterQuickfix extends BaseIntentionAction {
+  private final HaxeClassModel classModel;
+  private final HaxeFieldModel field;
+  private final Boolean generateGetter;
+
+  public CreateGetterSetterQuickfix(HaxeClassModel classModel, HaxeFieldModel field, Boolean generateGetter) {
+    this.classModel = classModel;
+    this.field = field;
+    this.generateGetter = generateGetter;
+  }
+
+  @NotNull
+  @Override
+  public String getText() {
+    return generateGetter ? "Generate getter" : "Generate setter";
+  }
+
+  @Nls
+  @NotNull
+  @Override
+  public String getFamilyName() {
+    return getText();
+  }
+
+  @Override
+  public boolean isAvailable(@NotNull Project project, Editor editor, PsiFile file) {
+    return true;
```

Comment:
This case is processed within annotator, quickfix registering only for not implemented setter or getter modifcator of property. 
e.g, for code like:
`public var variable(get, null):Type`

Only getter quickfix will be propsed.